app/models/fog_area.rb

```diff
@@ -1,8 +1,6 @@
 class FogArea < ApplicationRecord
-  BASIC_SVG_PATH = /M (?:-?\d+ ){2}(?:L (?:-?\d+ ){2})*Z/.freeze
-
   belongs_to :map
 
   validates :map, presence: true
-  validates :path, presence: true, format: { with: BASIC_SVG_PATH }
+  validates :path, presence: true
```

Comment:
Didn't think it was worth declaring `path` as a JSON serialized attribute as Rails never needs access to the JSON representation. But then we get the `safe_json` mess in the helper above...